manifests/security.pp

```diff
@@ -1,16 +1,25 @@
 #security for IRIDA instance
-class irida::security {
-
-
-  if $::facts['os']['selinux']['enabled'] {
-    include firewalld
-    firewalld_port { 'Open port 80 in the public zone':
+class irida::security(
+  Boolean $apache_use_ssl,
```

Comment:
Believe you need default value otherwise CI test will fail.  Probably default to false.

Reply:
Added default to false